src/Playwright/API/Generated/IBrowserType.cs

```diff
@@ -66,11 +66,6 @@ namespace Microsoft.Playwright
     /// </summary>
     public partial interface IBrowserType
     {
-        /// <summary><para>This methods attaches Playwright to an existing browser instance.</para></summary>
```

Comment:
Is this removal expected? I think it's better to not roll the release branch against the master branch in this case.

Reply:
I think this should be a draft for now. I was working on this then ended up fixing some other stuff before getting back to it.